docs/Migration-guide.md

```diff
@@ -505,4 +513,74 @@ This function can sometimes cause app hanging during window resize.
 Call instead the new `WindowUtils`.[UpdateBrowserSize](../api/WindowUtils.md#updatebrowsersize)
 function. Except when you use the `pywin32.py` example, in such case
 `WindowUtils.OnSize` must be called.
-See [Issue #464](../../../issues/464) for more details.
\ No newline at end of file
+See [Issue #464](../../../issues/464) for more details.
+
+## v123+ cef_types changed
+
+cef_types removed:
+- WOD_UNKNOWN,
+- WOD_CURRENT_TAB,
+- WOD_SINGLETON_TAB,
+- WOD_NEW_FOREGROUND_TAB,
+- WOD_NEW_BACKGROUND_TAB,
+- WOD_NEW_POPUP,
+- WOD_NEW_WINDOW,
+- WOD_SAVE_TO_DISK,
+- WOD_OFF_THE_RECORD,
+- WOD_IGNORE_ACTION
+- TID_FILE
+
+cef_types added:
+- CEF_WOD_UNKNOWN,
+- CEF_WOD_CURRENT_TAB,
+- CEF_WOD_SINGLETON_TAB,
+- CEF_WOD_NEW_FOREGROUND_TAB,
+- CEF_WOD_NEW_BACKGROUND_TAB,
+- CEF_WOD_NEW_POPUP,
+- CEF_WOD_NEW_WINDOW,
+- CEF_WOD_SAVE_TO_DISK,
+- CEF_WOD_OFF_THE_RECORD,
+- CEF_WOD_IGNORE_ACTION
+
+## v123+ Windows dpiaware change
+
+Per https://github.com/chromiumembedded/cef/issues/3452 windows high-dpi support is now enabled
+You shouldn't normally need to use DpiAware class 
+
+## v123+ plugin support is removed
+
+As cef has removed plugin support https://github.com/chromiumembedded/cef/commit/28c7f040016a0271ec2612cc5021599fb55a1054
+previous popular plugin such as flash, pdf would no longer work
+`OnBeforePluginLoad` handler is removed
+
+## v123+ cookie related code is changed
+
+- removed `CanGetCookies`
+- removed `CanSetCookie`
+- added `CanSendCookie`
+- added `CanSaveCookie`
```

Comment:
Let's mention CookieAccessFilter handler where new methods are, and also mention the old handler in which old cookie methods were.